lib/github-provider.js

```diff
@@ -68,9 +76,48 @@ function mapGithubCommitsSince(camp, githubApiUrl, githubAuth) {
     }));
 }
 
+//Github Release Date Integration
+function mapGithubReleaseDate(camp, githubApiUrl, githubAuth) {
+  camp.route(/^\/github\/release-date\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+    cache(function (data, match, sendBadge, request) {
+      const user = match[1];  // eg, microsoft
+      const repo = match[2];  // eg, vscode
+      const format = match[3];
+      const apiUrl = `${githubApiUrl}/repos/${user}/${repo}/releases/latest`;
+      const badgeData = getBadgeData('release date', data);
+      if (badgeData.template === 'social') {
+        badgeData.logo = getLogo('github', data);
+      }
+      githubAuth.request(request, apiUrl, {}, function (err, res, buffer) {
+        if (err != null) {
+          badgeData.text[1] = 'inaccessible';
+          sendBadge(format, badgeData);
+          return;
+        }
 
+        //github return 404 if repo not found or no release
+        if(res.statusCode === 404) {
+          badgeData.text[1] = 'not found';
+          sendBadge(format, badgeData);
```

Comment:
Is it good to return `not found` ? or just `invalid`??

Reply:
Yea, **no releases or repo not found** would be clearer than **invalid**.

Reply:
Big String ^^

Reply:
It's helpful though! It tells the user what could be wrong.